otx/core/data/adapter/detection_dataset_adapter.py

```diff
@@ -42,7 +42,7 @@ def get_otx_dataset(self) -> DatasetEntity:
                             and ann.type == DatumAnnotationType.polygon
                         ):
                             if self._is_normal_polygon(ann):
-                                shapes.append(self._get_polygon_entity(ann, image.width, image.height))
+                                shapes.append(self._get_polygon_entity(ann, image.width, image.height, 20))
```

Comment:
Is this intended?

Reply:
@eugene123tw, could you check?

Reply:
This will not help as it's sampling datumaro adapter. Is it right?

Reply:
> This will not help Geti scenario actually as it's sampling datumaro adapter. Is it right?

Yes, it couldn't help Geti scenario